CHANGELOG.md

```diff
@@ -50,6 +51,7 @@
 - [#3063](https://github.com/influxdata/telegraf/pull/3063): Add tls options to docker input.
 - [#2387](https://github.com/influxdata/telegraf/pull/2387): Add histogram aggregator plugin.
 - [#3080](https://github.com/influxdata/telegraf/pull/3080): Add zipkin input plugin.
+- [#2293](https://github.com/influxdata/telegraf/pull/2293): Add TCP listener for statsd input
```

Comment:
This line is added twice and there is a merge conflict.  Recently I have been adding the changelog updates separately to avoid these merge issues, so you can just remove all changes to this file.